synapse/models/risk.py

```diff
@@ -101,86 +101,86 @@ def getModelDefs(self):
+                        'doc': 'The reporter\'s assessed sophistication of the threat cluster.'}),
 
                     ('techniques', ('array', {'type': 'ou:technique', 'sorted': True, 'uniq': True}), {
                         'deprecated': True,
                         'doc': 'Deprecated for scalability. Please use -(uses)> ou:technique.'}),
 
                     ('merged:time', ('time', {}), {
-                        'doc': 'The time that this threat cluster was merged into another.'}),
+                        'doc': 'The time that the reporter merged this threat cluster into another.'}),
 
                     ('merged:isnow', ('risk:threat', {}), {
-                        'doc': 'The threat cluster that this cluster has been merged into.'}),
+                        'doc': 'The threat cluster that the reporter merged this cluster into.'}),
                 )),
                 ('risk:availability', {}, {}),
                 ('risk:tool:software:taxonomy', {}, ()),
                 ('risk:tool:software', {}, (
 
                     ('tag', ('syn:tag', {}), {
                         'ex': 'rep.mandiant.tabcteng',
-                        'doc': 'The tag used to annotate nodes that are part of the tool subgraph.'}),
+                        'doc': 'The tag used to annotate nodes that are associated with the tool.'}),
 
                     ('desc', ('str', {}), {
-                        'doc': "A description of the tool's use in threat activity."}),
+                        'doc': 'A description of the tool.'}),
 
                     ('type', ('risk:tool:software:taxonomy', {}), {
-                        'doc': 'An analyst specified taxonomy of software tool types.'}),
+                        'doc': 'A user-specified type for the tool.'}),
 
                     ('availability', ('risk:availability', {}), {
-                        'doc': 'The assessed availability of the tool.'}),
+                        'doc': 'The reporter\'s assessed availability of the tool.'}),
 
                     ('sophistication', ('meta:sophistication', {}), {
-                        'doc': 'The assessed sophistication of the tool.'}),
+                        'doc': 'The reporter\'s assessed sophistication of the tool.'}),
 
                     ('reporter', ('ou:org', {}), {
-                        'doc': 'The organization which reported the tool.'}),
+                        'doc': 'The organization reporting on the tool.'}),
 
                     ('reporter:name', ('ou:name', {}), {
-                        'doc': 'The name of the organization which reported the tool.'}),
+                        'doc': 'The name of the organization reporting on the tool.'}),
 
                     ('soft', ('it:prod:soft', {}), {
-                        'doc': 'The authoritative software family of the tool.'}),
+                        'doc': 'An authoritative software family for the tool.'}),
```

Comment:
This isn't "one of many" authoritative, so i think "the" is correct? Just like "the name of the organization" etc